app/models/task.py

```diff
@@ -3,4 +3,7 @@
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
```

Comment:
to make a relationship with the goal model you will use db.relationship inside the goal model like this 
`tasks = db.relationship('Task', backref='goal', lazy=True)`